src/cmd/usb-util.js

```diff
@@ -432,10 +467,15 @@ async function forEachUsbDevice(args, func, { dfuMode = false } = {}){
 				return Promise.resolve()
 					.then(async () => {
 						await executeWithUsbDevice({
-							args: { idOrName : usbDevice.id },
+							args: { idOrName : usbDevice.id, api: args.api, auth: args.auth },
 							func,
 							dfuMode
 						});
+						// This is particularly useful if the device handle is lost in the executed function
+						// Example: particle usb reset
+						if (usbDevice?.isOpen) {
+							await usbDevice.close();
+						}
```

Comment:
Still unclear to me why the handle is lost in the executeWithUsbDevice but this was needed to close the hanging handle

Reply:
There's a `device.close()` at end of `executeWithUsbDevice`. This seems extra